easybuild/easyconfigs/c/Circuitscape/Circuitscape-5.12.3-Julia-1.7.2.eb

```diff
@@ -0,0 +1,420 @@
+        'checksums': ['a69811f66cd1b7a6d91a0da30bf8a3ab796138615c4ada2d0f4ec044313b32e2'],
+    }),
+    ('DataAPI', '1.14.0', {
+        'source_urls': ['https://github.com/JuliaData/DataAPI.jl/archive/'],
+        'checksums': ['595e34053a6bfe9d504644a0df438560bc2676dba29c7175a733fb5300ce1ad4'],
+    }),
+    ('DataValueInterfaces', '1.0.0', {
+        'source_urls': ['https://github.com/queryverse/DataValueInterfaces.jl/archive/'],
+        'checksums': ['7f70be7cf1e49f7ce45bca8452193d08c2e75378f6b473205395c01010929825'],
+    }),
+    ('Tables', '1.10.0', {
+        'source_urls': ['https://github.com/JuliaData/Tables.jl/archive/'],
+        'checksums': ['4bb59d844830a0888de4ba3612484ce681e56b8571a6e4d6a605d442b3d1d0e4'],
+    }),
+    ('Graphics', '1.1.2', {
+        'source_urls': ['https://github.com/JuliaGraphics/Graphics.jl/archive/'],
+        'checksums': ['c28a6b29266f3c260616eef3d139184b267e0ae4d95ad89b379a5c973888bbb7'],
+    }),
+    ('PaddedViews', '0.5.11', {
+        'source_urls': ['https://github.com/JuliaArrays/PaddedViews.jl/archive/'],
+        'checksums': ['0506dc45cb371e6078fd055b8f21cd40755326e4b5c1b752e71489445e020cda'],
+    }),
+    ('MappedArrays', '0.4.1', {
+        'source_urls': ['https://github.com/JuliaArrays/MappedArrays.jl/archive/'],
+        'checksums': ['58ba4ef19d23737ffc2501e871545365fa1f6c5af84eef93549402a9f25fad56'],
+    }),
+    ('StackViews', '0.1.1', {
+        'source_urls': ['https://github.com/JuliaArrays/StackViews.jl/archive/'],
+        'checksums': ['8a8f008663c70ea9409263c9d2b78b0e246f9e6c700003e6a0e4dd5fe01558e9'],
+    }),
+    ('MosaicViews', '0.3.4', {
+        'source_urls': ['https://github.com/JuliaArrays/MosaicViews.jl/archive/'],
+        'sources': [{'download_filename': '7bb84df.tar.gz', 'filename': 'MosaicViews.v%(version)s.tar.gz'}],
+        'checksums': ['fd7fc8679417789b77130e847605fec2c9fe68a3de8a4b12f5c2612aee269273'],
+    }),
+    ('AbstractFFTs', '1.2.1', {
+        'source_urls': ['https://github.com/JuliaMath/AbstractFFTs.jl/archive/'],
+        'checksums': ['58abf90b037c618daeed89940c77ac129b5155b26429ac77643cc9c5db453363'],
+    }),
+    ('ImageCore', '0.9.4', {
+        'source_urls': ['https://github.com/JuliaImages/ImageCore.jl/archive/'],
+        'checksums': ['de5d244989891f47edc4f27bf8bf6557a6982e907f0874a62ad3480c9c333983'],
+    }),
+    ('ArchGDAL', '0.8.5', {
+        'source_urls': ['https://github.com/yeesian/ArchGDAL.jl/archive/'],
+        'checksums': ['e24ec57d88a3a07feac7db5b2501b8f06a1047c42a8eadcde9500e3601984065'],
+    }),
+    (name, version, {
+        'source_urls': ['https://github.com/Circuitscape/Circuitscape.jl/archive/'],
+        'checksums': ['1eb5b6f5711865792204677359c1b9d9247ae3f34af42f58dfe2ec4ad3dd2177'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['packages'],
+}
+
+sanity_check_commands = ["julia -e 'using Pkg;Pkg.test(\"Circuitscape\")'"]
```

Comment:
This makes sense even when using `JuliaBundle`, since it's running tests, keep it